eth/handler.go

```diff
@@ -162,7 +162,9 @@ func newHandler(config *handlerConfig) (*handler, error) {
 			log.Warn("Switch sync mode from full sync to snap sync")
 		}
 	} else {
-		if h.chain.CurrentBlock().Number.Uint64() > 0 {
+		blockNumber := h.chain.CurrentBlock().Number
+		// [Optimism] Allow snap sync from bedrock genesis block
```

Comment:
```suggestion
		// Allow snap sync from optimism bedrock genesis block
```

Adding `[Optimism]` prefix to comment is first introduced in this PR. Other optimism related comments do not have this prefix.